docs/revsets.md

```diff
@@ -51,19 +51,251 @@ The following operators are supported. `x` and `y` below can be any revset, not
 only symbols.
 
 * `x-`: Parents of `x`, can be empty.
+  <details>
+  <summary>Examples:</summary>
+  
+  Given this history:
+  ```
+  D
+  |\
+  | o C
+  | |
+  o | B
+  |/
+  o A
+  |
+  o root()
+  ```
+
+  `D-` ⇒ `{C,B}`
+
+  `B-` ⇒ `{A}`
+
+  `A-` ⇒ `{root()}`
+
+  `root()-` ⇒ `{}` (empty set)
+
+  `none()-` ⇒ `{}` (empty set)
+
+  `(D|A)-` ⇒ `{C,B,root()}`
+
+  `(C|B)-` ⇒ `{A}`
+  </details>
```

Comment:
I'd rather keep the operators list short, and add examples in separate section (by using the same graph.)

Even if all examples are folded, the list no longer fits in one screen.

Reply:
Done. Let me know what you think.